ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.17.yaml

```diff
@@ -319,25 +327,25 @@ tests:
       OCP_ARCH: arm64
       WORKER_INSTANCE_TYPE: m6g.xlarge
     workflow: openshift-e2e-aws-upi
-- as: ocp-e2e-aws-ovn-arm64-single-node
+- as: ocp-e2e-aws-ovn-sno-multi-a-a
   interval: 72h
   steps:
     cluster_profile: aws-2
     dependencies:
-      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
     env:
       BASE_DOMAIN: aws-2.ci.openshift.org
       OCP_ARCH: arm64
     observers:
       enable:
       - observers-resource-watch
     workflow: openshift-e2e-aws-single-node
-- as: ocp-e2e-aws-ovn-arm64-techpreview
+- as: ocp-e2e-aws-ovn-techpreview-multi-a-a
```

Comment:
here deployment style is probably techpreview, which comes before the payload again.